docs/user-guide.md

```diff
@@ -646,13 +646,16 @@ each account under that OU.
 deployment map as a target. If only the account number is in the deployment map
 the corresponding OU parameter file will not be referenced.
 
+
+
```

Comment:
Could you remove these empty lines, so there is only one around this code block?

```suggestion
```